tests/test_tokenization_t5.py

```diff
@@ -102,3 +106,25 @@ def test_full_tokenizer(self):
                 ".",
             ],
         )
+
+    def test_prepare_seq2seq_batch(self):
+        tokenizer = T5Tokenizer.from_pretrained("t5-small")
+        src_text = ["A long paragraph for summrization.", "Another paragraph for summrization."]
+        tgt_text = [
+            "Summary of the text.",
+            "Another summary.",
+        ]
+        expected_src_tokens = [71, 307, 8986, 21, 4505, 51, 52, 1707, 5, 1]
+        batch = tokenizer.prepare_seq2seq_batch(
+            src_text, tgt_texts=tgt_text, max_length=len(expected_src_tokens), return_tensors=FRAMEWORK
```

Comment:
More cases to test:
- test max_target_length kwarg and allow it to be passed through, affect decoder_input_ids.shape[1]
- empty tgt_texts
- empty src_texts -> Raises something

Reply:
Thanks, I will cover these cases.

Reply:
> empty tgt_texts

for this can I just check if `input_ids` and `attention_mask` are returned and no `decoder_input_ids` and `decoder_attention_mask` ?

Reply:
these tests look great now!